src/RemotePullManager.php

```diff
@@ -160,19 +179,16 @@ public function isAutopullNeeded(Remote $remote, $dry_run = FALSE) {
 
     // If autopull was never run or the intervals has been reached, we pull.
     if (!$last_autopull || ($last_autopull + $autopull_interval) < time()) {
-      // Don't process the pull on dry run.
-      if ($dry_run) {
-        return TRUE;
+      if (!$dry_run) {
```

Comment:
let's remove 1min / 60 from $autopull_interval so it gets reliably triggered from cron runs using the same interval.
in addition, instead of time() let's use the time service (dependency inject needed)
\Drupal::time()->getRequestTime();
https://www.drupal.org/node/2785211

Reply:
@fago There is no 1 minute $autopull_interval: https://github.com/drunomics/contentpool-client/pull/65/files the shortes interval is 5 min.

Reply:
I mean doing
`$autopull_interval = $autopull_interval - 60;`

problem is autopull 300 and cron 300 leads to the problem we have. it's not triggered every cron run since the diff might not be exactly 300 and won't be <300. So let's reduce the selected option by 60 to be sure.